### DISCUSSION TITLE
Why not let XMR seller set her own BTC redeem address?

### DISCUSSION THREAD

**Discussion by ghost** [comit-network/xmr-btc-swap#813] [Q&A] [answered]
is it technical or privacy issue?

**Comment by rishflab** [comment]
I think we can do this. There is no technical or privacy issues blocking this.

I will make an issue.

EDIT:

Had a chat with the team and this issue. The feature has been asked for in the past and It can be done but it is non trivial. 

We added an export bitcoin wallet command to the ASB in the release 0.9.0. Would that be sufficient for your needs?

You can export the internal wallet and manage it however you like.

**Reply by ghost** [reply]
> You can export the internal wallet and manage it however you like.

I've already done that but [Wallets Supporting Output Descriptors](https://outputdescriptors.org/) are very limited and not easy to use.

**Comment by delta1** [comment, marked as answer]
Done in #1259 - please try version 0.18 or newer
